pkg/util/retry.go

```diff
@@ -65,16 +67,16 @@ func IsErrorRetriable(err error) bool {
 	return ok
 }
 
-// BackoffWithDurationLimit returns backoff with a duration limit in 10 steps.
+// BackoffWithDurationLimit returns backoff with a duration limit.
 // Here is an example of the duration between steps:
 //
 //	1.055843837s, 2.085359785s, 4.229560375s, 8.324724174s, 16.295984061s,
 //	34.325711987s, 1m5.465642392s, 2m18.625713221s, 4m24.712222056s, 9m18.97652295s.
-func BackoffWithDurationLimit(duration time.Duration) wait.Backoff {
+func BackoffWithDurationLimit(duration time.Duration, steps int) wait.Backoff {
```

Comment:
https://github.com/GoogleContainerTools/kpt-config-sync/blob/v1.18.2-rc.2/pkg/parse/state.go#L100 can be consolidated with the same backoff function.

Reply:
done